build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     apply from: "https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-repositories.gradle", to: buildscript
     apply from: 'https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-cgp-version.gradle'
 
-    dependencies { classpath "com.synopsys.integration:common-gradle-plugin:1.12.0" }
+    dependencies { classpath "com.synopsys.integration:common-gradle-plugin:2.0.4" }
```

Comment:
It might be better to rely on the `managedCgpVersion` variable.

Example: https://github.com/blackducksoftware/blackduck-common/blob/master/build.gradle#L10